sklearn/multiclass.py

```diff
@@ -217,6 +218,7 @@ def fit(self, X, y):
 
         return self
 
+    @if_delegate_has_method(['_first_estimator', 'estimator'])
```

Comment:
needs testing? I think this should just be delegating to `estimator`. if the base estimator does not support `partial_fit`, nor can the fitted `_first_estimator`, no?